### DISCUSSION TITLE
How to config on_error with secrets/env vars?

### DISCUSSION THREAD

**Discussion by rahfar** [umputun/spot#177] [Q&A] [answered]
Hi, I want to add notification to telegram on error. Do not want to hardcode bot token in playbook, is it possible to use env vars or secrets in on_error field?

**Comment by umputun** [comment, marked as answer]
as of now - no. However there is no good reason not to support it, just smth I have never considered.

**Reply by umputun** [reply]
i have created  #178 for this feature request
